### ISSUE TITLE
AVT 1 - React on the Fixed SideNav & Fixed SideNav w/Icons UI Shell has DAP violations

### ISSUE THREAD

**Issue by snidersd** [carbon-design-system/carbon#3580]
## Environment
macOS Mojave version 10.14.5
Chrome Version 75.0.3770.100 (Official Build) (64-bit)
Carbon v10 - React
DAP - July 2019 Ruleset

## Detailed Description
Run DAP on the Fixed SideNav & Fixed SideNav w/Icons UI Shell. The following 2 violations:
![FixedSideNav](https://user-images.githubusercontent.com/21676914/62135983-d936ae00-b2a8-11e9-96e3-fbded4ca3de3.png)

**Comment by joshblack** [comment]
I believe this is related to our demo environment with two elements with `role="main"`. Feel free to comment and we can re-open if that's not the case 😄
